jumpcutter/__main__.py

```diff
@@ -54,6 +54,16 @@ def main() -> None:
         type=float,
         default=0.1,
     )
+    parser.add_argument(
+        "--preset",
+        "-p",
+        help="Sets the time that FFMPEG will take to compress the video. The slower,"
+        "the better the compression rate. Possibilities are: ``"ultrafast"``,
+        "``"superfast"``, ``"veryfast"``, ``"faster"`` (default), ``"fast"``,  ``"medium"``,
+        "``"slow"``, ``"slower"``, ``"veryslow"``, ``"placebo"``.",
```

Comment:
Can you use `choices=[...]` here? In this way, the help message can also be shorter.